posts/_markdown/2022_Q3.md

```diff
@@ -0,0 +1,263 @@
+[[Twitter]{.ul}](https://twitter.com/unitaryfund),[[LinkedIn]{.ul}](https://www.linkedin.com/company/unitary-fund/),
+and our [[Community
+Calendar]{.ul}](https://calendar.google.com/calendar/u/0/embed?src=c_mgqdq6hj2isi4d6h467kfqvg60@group.calendar.google.com).
+
+
+**New from UF**
+
+-   **Mitiq**
+
+    -   New Releases:
+         [[v0.17.1]{.ul}](https://github.com/unitaryfund/mitiq/releases/tag/v0.17.1),
+         [[v0.18.0]{.ul}](https://github.com/unitaryfund/mitiq/releases/tag/v0.18.0),
+         [[v0.19.0]{.ul}](https://github.com/unitaryfund/mitiq/releases/tag/v0.19.0)
+
+        -   In version 0.17.1, ​​The releases also include support for
+             the latest versions of Qiskit (0.37.1), Cirq (1.0.0), and
+             Pyquil (3.2.1).
+
+        -   V0.18.0 focused on review and approval of two RFCs, one for
+             Readout Error Mitigation (REM) and one for Identity
+             Insertion Noise Scaling.
+
+        -   Version 0.19 expanded support to the most recent versions of
+             Python: 3.8, 3.9 and 3.10, and is now compatible with
+             Numpy 1.21.6. We have also added new tools for applying
+             learning-based PEC, and introduced a function for learning
+             depolarizing noise representations from Clifford circuit
+             data.
+
+        -   Read more in our updated
+             [API-doc](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.pec.representations.biased_noise).
+
+        -   Special thanks to the external contributors
+             [\@yitchen-tim](https://github.com/yitchen-tim),
+             [\@amirebrahimi](https://github.com/amirebrahimi) and
+             [\@isaac-gs](https://github.com/isaac-gs)!
+
+    -   Talks:
+
+        -   September 22nd, 2022 - [IEEE Quantum
+             Week](https://qce.quantum.ieee.org/2022/), [Workshop on
+             quantum
+             IR](https://qce.quantum.ieee.org/2022/workshops-program/#alexandermccaskey),
+             *\"Mitiq: A cross-platform error mitigation toolkit\"*.
+             \@andreamari
+
+        -   September 13th, 2022 - [Quantum Africa 6
+             Conference](https://qa.eaifr.org/), *\"Quantum Error
+             Mitigation: An open-source software approach\"*.
+             \@nathanshammah
+
+        -   August 15th, 2022 - [WOMANIUM
+             QUANTUM](https://www.youtube.com/watch?v=3LAHjmSuyO8&list=PL-VMs2BCTI_lklMMfY4iMdETT19rgZe5o)
+             \@Misty-W
+
+    -   The [[Mitiq whitepaper]{.ul}](https://arxiv.org/abs/2009.04417)
+         was published in *Quantum **6**, 774 (2022)*
+
+-   [**[Metriq]{.ul}**](https://metriq.info/)
+
+    -   New Releases:
+         [[v.0.6.0]{.ul}](https://github.com/unitaryfund/metriq-app/releases/tag/v0.6.0),
+         [[v.0.6.1]{.ul}](https://github.com/unitaryfund/metriq-app/releases/tag/v0.6.1)
+
+        -   Version 0.6.0 focuses on UX upgrades, including new member
+             self-curation by granular subscription. Single-page
```

Comment:
I don't know what this sentence means. What is "new member self-curation by granular subscription"?